dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -317,7 +317,10 @@ def test_categorical_diff(self):
         df_categorical = pd.Series(["y", "maybe", "y", "y", "n", "n", "maybe"])
         profile2 = CategoricalColumn(df_categorical.name)
         profile2.update(df_categorical)
-        
+
+        # chi2-statistic = sum((observed-expected)^2/expected for each category in each column)
+        # df = categories - 1
+        # p-value found through using chi2 CDF
```

Comment:
remove?

Reply:
i think this comment maybe needs more clarity which will explain what's happening below.